petermwash/JavaDevsNairobi
# #155820645 Create layout for the DetailActivity

#### What does this PR do?
- Create layout file for DetailActivity.

#### Description of Task to be completed?
- Create the styling for the DetailActivity.
- Launch the DetailActivity from the MainActivity

#### How should this be manually tested?
- Clone the project, `git clone https://github.com/petermwash/JavaDevsNairobi.git`.
- On the terminal, cd into `JavaDevsNairobi` directory.
- Checkout to the `ft-create-layout-file-for-DetailActivity-155820645`.
- Now you can run the app and on emulator when the main activity launches, click on the view detail card to launch the DetailActivity.
- You should be able to see a screen as the one shown on the screenshots below.

#### Any background context you want to provide?
- I have also updated the content of the the README on this PR to capture my reasons for choosing the Justinmind wire-framing tool.

#### What are the relevant pivotal tracker stories?
[#155820645](https://www.pivotaltracker.com/story/show/155820645)
[#155820624](https://www.pivotaltracker.com/story/show/155820624)

#### Screenshots (if appropriate)
- DetailActivity layout.
<img width="736" alt="screen shot 2018-10-19 at 00 35 43" src="https://user-images.githubusercontent.com/27728464/47209544-72fa2380-d345-11e8-9866-f77e61adffd4.png">


#### Questions:
No
